server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java

```diff
@@ -67,8 +67,12 @@ public NettyTransport(InetSocketAddress address, ProtocolServerConfiguration con
       this.configuration = configuration;
 
       // Need to initialize these in constructor since they require configuration
-      masterGroup = buildEventLoop(1, new DefaultThreadFactory(threadNamePrefix + "-ServerMaster"));
-      ioGroup = buildEventLoop(configuration.ioThreads(), new DefaultThreadFactory(threadNamePrefix + "-ServerIO"));
+      masterGroup = serverEventLoop(1, new DefaultThreadFactory(threadNamePrefix + "-ServerMaster"));
+      if (shouldShutdownIOGroup = cacheManager == null) {
```

Comment:
Please move the assignment to its own line, have you been reading lots JDK code lately? :)

Reply:
Okay